library/std/src/io/util.rs

```diff
@@ -58,6 +60,26 @@ impl BufRead for Empty {
     fn consume(&mut self, _n: usize) {}
 }
 
+#[stable(feature = "empty_seek", since = "1.49.0")]
```

Comment:
This should be 1.50, 1.49 is currently in beta.
```suggestion
#[stable(feature = "empty_seek", since = "1.50.0")]
```